xapian-letor/api/featuremanager_internal.cc

```diff
@@ -130,7 +128,6 @@ FeatureManager::Internal::load_relevance(const std::string & qrel_file)
         str = strtok(NULL, " ,.-");
         }
 
-        qrel1.insert(make_pair(token[0], Map1()));
```

Comment:
Removed this as insertion into inner map id being done in the following line. Is there something wrong?